content-src/components/SectionMenu/SectionMenu.jsx

```diff
@@ -51,15 +63,19 @@ export class _SectionMenu extends React.PureComponent {
         const { action, id, type, userEvent } = option;
         if (!type && id) {
           option.onClick = () => {
-            props.dispatch(action);
-            if (userEvent) {
-              props.dispatch(
-                ac.UserEvent({
-                  event: userEvent,
-                  source: props.source,
-                })
-              );
+            const addingElements =
```

Comment:
nit: more readable if boolean value variable name starting with is, has... We can rename to something like hasAddEvent